### ISSUE TITLE
Ubuntu 22.04 armv7l: `cc1: error: '-mfloat-abi=hard': selected architecture lacks an FPU`

### ISSUE THREAD

**Issue by cattokomo** [openssl/openssl#19557]
Hello 👋

I have a problem while building a program that uses OpenSSL.

The log of the `openssl-build-err.log`:
```
make[4]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
cc1: error: '-mfloat-abi=hard': selected architecture lacks an FPU
make[5]: *** [Makefile:666: crypto/aes/aes_cbc.o]
make[4]: *** [Makefile:171: all] Error 2
```

The command that reproduce the problem:
```
$ cd luvi-master
$ make regular -j8
cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Release -DWithSharedLibluv=OFF -DWithPackageSH=ON -DWithPackageTGZ=ON -DWithPackageTBZ2=ON -DWithOpenSSL=ON -DWithSharedOpenSSL=OFF -DWithPCRE=ON -DWithLPEG=ON -DWithSharedPCRE=OFF
CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- The C compiler identification is GNU 11.3.0
-- The ASM compiler identification is GNU
-- Found assembler: /usr/bin/cc
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Found luvi git version: v2.14.0-4-g8eae450
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
CMake Deprecation Warning at deps/luv/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- Performing Test UV_LINT_W4
-- Performing Test UV_LINT_W4 - Failed
-- Performing Test UV_LINT_NO_UNUSED_PARAMETER_MSVC
-- Performing Test UV_LINT_NO_UNUSED_PARAMETER_MSVC - Failed
-- Performing Test UV_LINT_NO_CONDITIONAL_CONSTANT_MSVC
-- Performing Test UV_LINT_NO_CONDITIONAL_CONSTANT_MSVC - Failed
-- Performing Test UV_LINT_NO_NONSTANDARD_MSVC
-- Performing Test UV_LINT_NO_NONSTANDARD_MSVC - Failed
-- Performing Test UV_LINT_NO_NONSTANDARD_EMPTY_TU_MSVC
-- Performing Test UV_LINT_NO_NONSTANDARD_EMPTY_TU_MSVC - Failed
-- Performing Test UV_LINT_NO_NONSTANDARD_FILE_SCOPE_MSVC
-- Performing Test UV_LINT_NO_NONSTANDARD_FILE_SCOPE_MSVC - Failed
-- Performing Test UV_LINT_NO_NONSTANDARD_NONSTATIC_DLIMPORT_MSVC
-- Performing Test UV_LINT_NO_NONSTANDARD_NONSTATIC_DLIMPORT_MSVC - Failed
-- Performing Test UV_LINT_NO_HIDES_LOCAL
-- Performing Test UV_LINT_NO_HIDES_LOCAL - Failed
-- Performing Test UV_LINT_NO_HIDES_PARAM
-- Performing Test UV_LINT_NO_HIDES_PARAM - Failed
-- Performing Test UV_LINT_NO_HIDES_GLOBAL
-- Performing Test UV_LINT_NO_HIDES_GLOBAL - Failed
-- Performing Test UV_LINT_NO_CONDITIONAL_ASSIGNMENT_MSVC
-- Performing Test UV_LINT_NO_CONDITIONAL_ASSIGNMENT_MSVC - Failed
-- Performing Test UV_LINT_NO_UNSAFE_MSVC
-- Performing Test UV_LINT_NO_UNSAFE_MSVC - Failed
-- Performing Test UV_LINT_WALL
-- Performing Test UV_LINT_WALL - Success
-- Performing Test UV_LINT_NO_UNUSED_PARAMETER
-- Performing Test UV_LINT_NO_UNUSED_PARAMETER - Success
-- Performing Test UV_LINT_STRICT_PROTOTYPES
-- Performing Test UV_LINT_STRICT_PROTOTYPES - Success
-- Performing Test UV_LINT_EXTRA
-- Performing Test UV_LINT_EXTRA - Success
-- Performing Test UV_LINT_UTF8_MSVC
-- Performing Test UV_LINT_UTF8_MSVC - Failed
-- Performing Test UV_F_STRICT_ALIASING
-- Performing Test UV_F_STRICT_ALIASING - Success
-- summary of build options:
    Install prefix:  /usr/local
    Target system:   Linux
    Compiler:
      C compiler:    /usr/bin/cc (GNU)
      CFLAGS:

-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of void*
-- Check size of void* - done
-- Looking for dlopen
-- Looking for dlopen - found
-- Looking for sin in m
-- Looking for sin in m - found
-- Performing Test LJ_TARGET_X64
-- Performing Test LJ_TARGET_X64 - Failed
-- Performing Test LJ_TARGET_X86
-- Performing Test LJ_TARGET_X86 - Failed
-- Performing Test LJ_TARGET_ARM
-- Performing Test LJ_TARGET_ARM - Success
-- LuaJIT Target: arm
-- LuaJIT target arm
-- Performing Test LJ_LE_1
-- Performing Test LJ_LE_1 - Success
-- Performing Test LJ_ARCH_BITS_64
-- Performing Test LJ_ARCH_BITS_64 - Failed
-- Performing Test LJ_HASJIT_1
-- Performing Test LJ_HASJIT_1 - Success
-- Performing Test LJ_HASFFI_1
-- Performing Test LJ_HASFFI_1 - Success
-- Performing Test LJ_DUALNUM_1
-- Performing Test LJ_DUALNUM_1 - Success
-- Performing Test LJ_ARCH_HASFPU_1
-- Performing Test LJ_ARCH_HASFPU_1 - Success
-- Performing Test LJ_ABI_SOFTFP_1
-- Performing Test LJ_ABI_SOFTFP_1 - Failed
-- Performing Test LJ_NO_UNWIND_1
-- Performing Test LJ_NO_UNWIND_1 - Failed
-- Checking if the toolchain always generates unwind tables
-- Toolchain does not generate unwind tables
-- Looking for sin in m
-- Looking for sin in m - found
Enabling Static OpenSSL
-- The CXX compiler identification is GNU 11.3.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Deprecation Warning at deps/pcre/CMakeLists.txt:73 (CMAKE_MINIMUM_REQUIRED):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- Could NOT find BZip2 (missing: BZIP2_LIBRARIES BZIP2_INCLUDE_DIR)
-- Found ZLIB: /usr/lib/arm-linux-gnueabihf/libz.so (found version "1.2.11")
-- Found Readline: /usr/include
-- Could not find OPTIONAL package Editline
-- Looking for dirent.h
-- Looking for dirent.h - found
-- Looking for inttypes.h
-- Looking for inttypes.h - found
-- Looking for sys/stat.h
-- Looking for sys/stat.h - found
-- Looking for unistd.h
-- Looking for unistd.h - found
-- Looking for windows.h
-- Looking for windows.h - not found
-- Looking for C++ include type_traits.h
-- Looking for C++ include type_traits.h - not found
-- Looking for C++ include bits/type_traits.h
-- Looking for C++ include bits/type_traits.h - not found
-- Looking for bcopy
-- Looking for bcopy - found
-- Looking for memmove
-- Looking for memmove - found
-- Looking for strerror
-- Looking for strerror - found
-- Looking for strtoll
-- Looking for strtoll - found
-- Looking for strtoq
-- Looking for strtoq - found
-- Looking for _strtoi64
-- Looking for _strtoi64 - not found
-- Check size of long long
-- Check size of long long - done
-- Check size of unsigned long long
-- Check size of unsigned long long - done
--
--
-- PCRE configuration summary:
--
--   Install prefix .................. : /usr/local
--   C compiler ...................... : /usr/bin/cc
--   C++ compiler .................... : /usr/bin/c++
--   C compiler flags ................ : -O3 -DNDEBUG
--   C++ compiler flags .............. : -O3 -DNDEBUG
--
--   Build 8 bit PCRE library ........ : ON
--   Build 16 bit PCRE library ....... : OFF
--   Build 32 bit PCRE library ....... : OFF
--   Build C++ library ............... : OFF
--   Enable JIT compiling support .... : OFF
--   Enable UTF support .............. : ON
--   Unicode properties .............. : OFF
--   Newline char/sequence ........... : LF
--   \R matches only ANYCRLF ......... : OFF
--   EBCDIC coding ................... : OFF
--   EBCDIC coding with NL=0x25 ...... : OFF
--   Rebuild char tables ............. : OFF
--   No stack recursion .............. : OFF
--   POSIX mem threshold ............. : 10
--   Internal link size .............. : 2
--   Parentheses nest limit .......... : 250
--   Match limit ..................... : 150000
--   Match limit recursion ........... : MATCH_LIMIT
--   Build shared libs ............... : OFF
--   Build static libs ............... : ON
--   Build pcregrep .................. : OFF
--   Enable JIT in pcregrep .......... : ON
--   Buffer size for pcregrep ........ : 20480
--   Build tests (implies pcretest  .. : OFF
--                and pcregrep)
--   Link pcregrep with libz ......... : ON
--   Link pcregrep with libbz2 ....... : Library not found
--   Link pcretest with libeditline .. : Library not found
--   Link pcretest with libreadline .. : ON
--   Support Valgrind .................: OFF
--   Support coverage .................:
--
Enabling Static PCRE
-- Configuring done
-- Generating done
-- Build files have been written to: /home/UrNightmaree/.L/repo/luvi-master/build
$ make -j8
cmake --build build -- -j8
gmake[1]: warning: -j8 forced in submake: resetting jobserver mode.
gmake[1]: Entering directory '/home/UrNightmaree/.L/repo/luvi-master/build'
gmake[2]: Entering directory '/home/UrNightmaree/.L/repo/luvi-master/build'
gmake[3]: Entering directory '/home/UrNightmaree/.L/repo/luvi-master/build'
gmake[3]: Entering directory '/home/UrNightmaree/.L/repo/luvi-master/build'
gmake[3]: Entering directory '/home/UrNightmaree/.L/repo/luvi-master/build'
gmake[3]: Entering directory '/home/UrNightmaree/.L/repo/luvi-master/build'
gmake[3]: Entering directory '/home/UrNightmaree/.L/repo/luvi-master/build'
gmake[3]: Leaving directory '/home/UrNightmaree/.L/repo/luvi-master/build'
gmake[3]: Leaving directory '/home/UrNightmaree/.L/repo/luvi-master/build'
gmake[3]: Leaving directory '/home/UrNightmaree/.L/repo/luvi-master/build'
gmake[3]: Leaving directory '/home/UrNightmaree/.L/repo/luvi-master/build'
gmake[3]: Entering directory '/home/UrNightmaree/.L/repo/luvi-master/build'
gmake[3]: Entering directory '/home/UrNightmaree/.L/repo/luvi-master/build'
gmake[3]: Entering directory '/home/UrNightmaree/.L/repo/luvi-master/build'
gmake[3]: Leaving directory '/home/UrNightmaree/.L/repo/luvi-master/build'
gmake[3]: Entering directory '/home/UrNightmaree/.L/repo/luvi-master/build'
gmake[3]: Entering directory '/home/UrNightmaree/.L/repo/luvi-master/build'
[  1%] Building C object CMakeFiles/LPEGLIB.dir/deps/lpeg/lpcode.c.o
[  1%] Building C object CMakeFiles/LPEGLIB.dir/deps/lpeg/lpprint.c.o
[  2%] Building C object deps/pcre/CMakeFiles/pcre.dir/pcre_byte_order.c.o
[  2%] Building C object CMakeFiles/LPEGLIB.dir/deps/lpeg/lpcap.c.o
[  3%] Building C object CMakeFiles/LPEGLIB.dir/deps/lpeg/lptree.c.o
[  4%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/fs-poll.c.o
[  6%] Creating directories for 'openssl'
[  6%] Building C object deps/luv/CMakeFiles/minilua.dir/deps/luajit/src/host/minilua.c.o
In file included from /home/UrNightmaree/.L/repo/luvi-master/deps/lpeg/lptree.c:14:
/home/UrNightmaree/.L/repo/luvi-master/deps/lpeg/lptypes.h:38: warning: "luaL_newlib" redefined
   38 | #define luaL_newlib(L,f)        luaL_register(L,"lpeg",f)
      |
In file included from /home/UrNightmaree/.L/repo/luvi-master/deps/lpeg/lptree.c:12:
/home/UrNightmaree/.L/repo/luvi-master/deps/luv/deps/luajit/src/lauxlib.h:125: note: this is the location of the previous definition
  125 | #define luaL_newlib(L, l)       (luaL_newlibtable(L, l), luaL_setfuncs(L, l, 0))
      |
In file included from /home/UrNightmaree/.L/repo/luvi-master/deps/lpeg/lpcap.h:9,
                 from /home/UrNightmaree/.L/repo/luvi-master/deps/lpeg/lpcap.c:9:
/home/UrNightmaree/.L/repo/luvi-master/deps/lpeg/lptypes.h:38: warning: "luaL_newlib" redefined
   38 | #define luaL_newlib(L,f)        luaL_register(L,"lpeg",f)
      |
In file included from /home/UrNightmaree/.L/repo/luvi-master/deps/lpeg/lpcap.c:7:
/home/UrNightmaree/.L/repo/luvi-master/deps/luv/deps/luajit/src/lauxlib.h:125: note: this is the location of the previous definition
  125 | #define luaL_newlib(L, l)       (luaL_newlibtable(L, l), luaL_setfuncs(L, l, 0))
      |
[  6%] Building C object deps/pcre/CMakeFiles/pcre.dir/pcre_chartables.c.o
[  6%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/idna.c.o
[  7%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/inet.c.o
[  8%] Performing download step (download, verify and extract) for 'openssl'
-- Downloading...
   dst='/home/UrNightmaree/.L/repo/luvi-master/build/openssl/src/openssl-1.1.1m.tar.gz'
   timeout='none'
   inactivity timeout='none'
-- Using src='https://www.openssl.org/source/openssl-1.1.1m.tar.gz'
[  9%] Building C object deps/pcre/CMakeFiles/pcre.dir/pcre_compile.c.o
[  9%] Building C object CMakeFiles/LPEGLIB.dir/deps/lpeg/lpvm.c.o
-- [download 0% complete]
-- [download 1% complete]
-- [download 2% complete]
-- [download 3% complete]
-- [download 4% complete]
-- [download 5% complete]
-- [download 6% complete]
-- [download 7% complete]
-- [download 8% complete]
-- [download 9% complete]
-- [download 10% complete]
-- [download 11% complete]
-- [download 12% complete]
-- [download 13% complete]
-- [download 14% complete]
-- [download 15% complete]
-- [download 16% complete]
-- [download 17% complete]
-- [download 18% complete]
-- [download 19% complete]
-- [download 20% complete]
-- [download 21% complete]
-- [download 22% complete]
-- [download 23% complete]
In file included from /home/UrNightmaree/.L/repo/luvi-master/deps/lpeg/lpcap.h:9,
                 from /home/UrNightmaree/.L/repo/luvi-master/deps/lpeg/lpvm.c:13:
/home/UrNightmaree/.L/repo/luvi-master/deps/lpeg/lptypes.h:38: warning: "luaL_newlib" redefined
   38 | #define luaL_newlib(L,f)        luaL_register(L,"lpeg",f)
      |
In file included from /home/UrNightmaree/.L/repo/luvi-master/deps/lpeg/lpvm.c:11:
/home/UrNightmaree/.L/repo/luvi-master/deps/luv/deps/luajit/src/lauxlib.h:125: note: this is the location of the previous definition
  125 | #define luaL_newlib(L, l)       (luaL_newlibtable(L, l), luaL_setfuncs(L, l, 0))
      |
-- [download 24% complete]
-- [download 25% complete]
-- [download 26% complete]
-- [download 27% complete]
-- [download 28% complete]
-- [download 29% complete]
-- [download 30% complete]
-- [download 31% complete]
-- [download 32% complete]
-- [download 33% complete]
-- [download 34% complete]
-- [download 35% complete]
-- [download 36% complete]
-- [download 37% complete]
-- [download 38% complete]
-- [download 39% complete]
-- [download 40% complete]
-- [download 41% complete]
-- [download 42% complete]
-- [download 43% complete]
-- [download 44% complete]
-- [download 45% complete]
-- [download 46% complete]
-- [download 47% complete]
[  9%] Building C object deps/pcre/CMakeFiles/pcre.dir/pcre_config.c.o
-- [download 48% complete]
-- [download 49% complete]
-- [download 50% complete]
[  9%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/random.c.o
-- [download 51% complete]
-- [download 52% complete]
-- [download 53% complete]
-- [download 54% complete]
-- [download 55% complete]
-- [download 56% complete]
-- [download 57% complete]
-- [download 58% complete]
-- [download 59% complete]
-- [download 60% complete]
-- [download 61% complete]
-- [download 62% complete]
In file included from /home/UrNightmaree/.L/repo/luvi-master/deps/lpeg/lpcode.c:12:
/home/UrNightmaree/.L/repo/luvi-master/deps/lpeg/lptypes.h:38: warning: "luaL_newlib" redefined
   38 | #define luaL_newlib(L,f)        luaL_register(L,"lpeg",f)
      |
-- [download 63% complete]
In file included from /home/UrNightmaree/.L/repo/luvi-master/deps/lpeg/lpcode.c:10:
/home/UrNightmaree/.L/repo/luvi-master/deps/luv/deps/luajit/src/lauxlib.h:125: note: this is the location of the previous definition
  125 | #define luaL_newlib(L, l)       (luaL_newlibtable(L, l), luaL_setfuncs(L, l, 0))
      |
-- [download 64% complete]
-- [download 65% complete]
-- [download 66% complete]
-- [download 67% complete]
-- [download 68% complete]
-- [download 69% complete]
-- [download 70% complete]
-- [download 71% complete]
-- [download 72% complete]
-- [download 73% complete]
-- [download 74% complete]
-- [download 75% complete]
-- [download 76% complete]
-- [download 77% complete]
-- [download 78% complete]
-- [download 79% complete]
-- [download 80% complete]
-- [download 81% complete]
[ 11%] Building C object deps/pcre/CMakeFiles/pcre.dir/pcre_dfa_exec.c.o
-- [download 82% complete]
-- [download 83% complete]
-- [download 84% complete]
-- [download 85% complete]
-- [download 86% complete]
-- [download 87% complete]
-- [download 88% complete]
-- [download 89% complete]
-- [download 90% complete]
-- [download 91% complete]
-- [download 92% complete]
-- [download 93% complete]
-- [download 94% complete]
-- [download 95% complete]
-- [download 96% complete]
-- [download 97% complete]
-- [download 98% complete]
-- [download 99% complete]
-- [download 100% complete]
-- verifying file...
       file='/home/UrNightmaree/.L/repo/luvi-master/build/openssl/src/openssl-1.1.1m.tar.gz'
-- Downloading... done
[ 11%] Building C object deps/pcre/CMakeFiles/pcre.dir/pcre_exec.c.o
-- extracting...
     src='/home/UrNightmaree/.L/repo/luvi-master/build/openssl/src/openssl-1.1.1m.tar.gz'
     dst='/home/UrNightmaree/.L/repo/luvi-master/build/openssl/src/openssl'
-- extracting... [tar xfz]
[ 12%] Linking C static library libLPEGLIB.a
gmake[3]: Leaving directory '/home/UrNightmaree/.L/repo/luvi-master/build'
[ 12%] Built target LPEGLIB
[ 13%] Building C object deps/pcre/CMakeFiles/pcre.dir/pcre_fullinfo.c.o
[ 14%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/strscpy.c.o
[ 14%] Building C object deps/pcre/CMakeFiles/pcre.dir/pcre_get.c.o
[ 14%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/strtok.c.o
[ 16%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/threadpool.c.o
[ 17%] Linking C executable minilua
[ 17%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/timer.c.o
gmake[3]: Leaving directory '/home/UrNightmaree/.L/repo/luvi-master/build'
[ 18%] Building C object deps/pcre/CMakeFiles/pcre.dir/pcre_globals.c.o
[ 18%] Built target minilua
[ 18%] Building C object deps/pcre/CMakeFiles/pcre.dir/pcre_jit_compile.c.o
gmake[3]: Entering directory '/home/UrNightmaree/.L/repo/luvi-master/build'
[ 18%] Generating buildvm_arch.h
gmake[3]: Leaving directory '/home/UrNightmaree/.L/repo/luvi-master/build'
gmake[3]: Entering directory '/home/UrNightmaree/.L/repo/luvi-master/build'
[ 19%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/uv-common.c.o
[ 20%] Building C object deps/luv/CMakeFiles/buildvm.dir/deps/luajit/src/host/buildvm.c.o
[ 20%] Building C object deps/luv/CMakeFiles/buildvm.dir/deps/luajit/src/host/buildvm_asm.c.o
[ 22%] Building C object deps/luv/CMakeFiles/buildvm.dir/deps/luajit/src/host/buildvm_fold.c.o
[ 23%] Building C object deps/pcre/CMakeFiles/pcre.dir/pcre_maketables.c.o
[ 23%] Building C object deps/pcre/CMakeFiles/pcre.dir/pcre_newline.c.o
[ 23%] Building C object deps/luv/CMakeFiles/buildvm.dir/deps/luajit/src/host/buildvm_lib.c.o
[ 23%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/uv-data-getter-setters.c.o
[ 24%] Building C object deps/pcre/CMakeFiles/pcre.dir/pcre_ord2utf8.c.o
[ 25%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/version.c.o
[ 27%] Building C object deps/luv/CMakeFiles/buildvm.dir/deps/luajit/src/host/buildvm_peobj.c.o
[ 27%] Building C object deps/pcre/CMakeFiles/pcre.dir/pcre_refcount.c.o
[ 27%] Linking C executable buildvm
[ 27%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/async.c.o
gmake[3]: Leaving directory '/home/UrNightmaree/.L/repo/luvi-master/build'
[ 27%] Built target buildvm
[ 28%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/core.c.o
gmake[3]: Entering directory '/home/UrNightmaree/.L/repo/luvi-master/build'
[ 28%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/dl.c.o
[ 28%] Generating vmdef.lua
[ 29%] Building C object deps/pcre/CMakeFiles/pcre.dir/pcre_string_utils.c.o
[ 29%] Generating lj_bcdef.h
[ 30%] Generating lj_ffdef.h
[ 32%] Generating lj_folddef.h
[ 33%] Generating lj_libdef.h
[ 34%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/fs.c.o
[ 34%] Building C object deps/pcre/CMakeFiles/pcre.dir/pcre_study.c.o
[ 34%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/getaddrinfo.c.o
[ 34%] Generating lj_recdef.h
[ 34%] Generating lj_vm.S
Scanning dependencies of target luajit-5.1
gmake[3]: Leaving directory '/home/UrNightmaree/.L/repo/luvi-master/build'
[ 35%] Building C object deps/pcre/CMakeFiles/pcre.dir/pcre_tables.c.o
[ 37%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/getnameinfo.c.o
gmake[3]: Entering directory '/home/UrNightmaree/.L/repo/luvi-master/build'
[ 38%] Building C object deps/luv/CMakeFiles/luajit-5.1.dir/deps/luajit/src/ljamalg.c.o
[ 38%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/loop-watcher.c.o
[ 38%] Building ASM object deps/luv/CMakeFiles/luajit-5.1.dir/lj_vm.S.o
[ 38%] Building C object deps/pcre/CMakeFiles/pcre.dir/pcre_ucd.c.o
[ 39%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/loop.c.o
[ 40%] Building C object deps/pcre/CMakeFiles/pcre.dir/pcre_valid_utf8.c.o              [ 40%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/pipe.c.o
[ 40%] Building C object deps/pcre/CMakeFiles/pcre.dir/pcre_version.c.o
[ 41%] Building C object deps/pcre/CMakeFiles/pcre.dir/pcre_xclass.c.o
[ 43%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/poll.c.o
[ 43%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/process.c.o
[ 44%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/random-devurandom.c.o
[ 45%] Linking C static library libpcre.a
gmake[3]: Leaving directory '/home/UrNightmaree/.L/repo/luvi-master/build'              [ 45%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/signal.c.o
[ 45%] Built target pcre
gmake[3]: Entering directory '/home/UrNightmaree/.L/repo/luvi-master/build'
[ 46%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/stream.c.o
[ 46%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/tcp.c.o       gmake[3]: Leaving directory '/home/UrNightmaree/.L/repo/luvi-master/build'
gmake[3]: Entering directory '/home/UrNightmaree/.L/repo/luvi-master/build'
[ 48%] Building C object CMakeFiles/lrexlib.dir/deps/lrexlib/src/common.c.o
gmake[3]: Entering directory '/home/UrNightmaree/.L/repo/luvi-master/build'
gmake[3]: Leaving directory '/home/UrNightmaree/.L/repo/luvi-master/build'
gmake[3]: Entering directory '/home/UrNightmaree/.L/repo/luvi-master/build'
[ 48%] Building C object deps/pcre/CMakeFiles/pcreposix.dir/pcreposix.c.o
[ 48%] Building C object CMakeFiles/lrexlib.dir/deps/lrexlib/src/pcre/lpcre.c.o
[ 49%] Building C object CMakeFiles/lrexlib.dir/deps/lrexlib/src/pcre/lpcre_f.c.o
[ 50%] Linking C static library libpcreposix.a
[ 51%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/thread.c.o
gmake[3]: Leaving directory '/home/UrNightmaree/.L/repo/luvi-master/build'
[ 51%] Built target pcreposix
[ 51%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/tty.c.o
[ 53%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/udp.c.o
[ 53%] Linking C static library liblrexlib.a
gmake[3]: Leaving directory '/home/UrNightmaree/.L/repo/luvi-master/build'
[ 53%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/proctitle.c.o
[ 53%] Built target lrexlib
[ 54%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/linux-core.c.o
-- extracting... [analysis]
-- extracting... [rename]
-- extracting... [clean up]
-- extracting... done
[ 54%] No update step for 'openssl'
[ 54%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/linux-inotify.c.o
[ 54%] No patch step for 'openssl'
[ 55%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/linux-syscalls.c.o
[ 55%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/procfs-exepath.c.o
[ 55%] Performing configure step for 'openssl'
Operating system: armv8l-whatever-linux2
[ 56%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/random-getrandom.c.o
[ 56%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/random-sysctl-linux.c.o
Configuring OpenSSL version 1.1.1m (0x101010dfL) for linux-armv4
Using os-specific seed configuration
[ 58%] Building C object deps/luv/deps/libuv/CMakeFiles/uv_a.dir/src/unix/epoll.c.o
[ 59%] Linking C static library libuv_a.a
gmake[3]: Leaving directory '/home/UrNightmaree/.L/repo/luvi-master/build'
[ 59%] Built target uv_a
gmake[3]: Entering directory '/home/UrNightmaree/.L/repo/luvi-master/build'
gmake[3]: Leaving directory '/home/UrNightmaree/.L/repo/luvi-master/build'
gmake[3]: Entering directory '/home/UrNightmaree/.L/repo/luvi-master/build'
[ 60%] Building C object deps/luv/CMakeFiles/libluv_a.dir/src/luv.c.o
[ 60%] Linking C static library libluv.a
gmake[3]: Leaving directory '/home/UrNightmaree/.L/repo/luvi-master/build'
[ 60%] Built target libluv_a
Creating configdata.pm
Creating Makefile

**********************************************************************
***                                                                ***
***   OpenSSL has been successfully configured                     ***
***                                                                ***
***   If you encounter a problem while building, please open an    ***
***   issue on GitHub <https://github.com/openssl/openssl/issues>  ***
***   and include the output from the following command:           ***
***                                                                ***
***       perl configdata.pm --dump                                ***
***                                                                ***
***   (If you are new to OpenSSL, you might want to consult the    ***
***   'Troubleshooting' section in the INSTALL file first)         ***
***                                                                ***
**********************************************************************
[ 61%] Performing build step for 'openssl'
CMake Error at /home/UrNightmaree/.L/repo/luvi-master/build/openssl/src/openssl-stamp/openssl-build-Release.cmake:49 (message):
  Command failed: 2

   'make'

  See also

    /home/UrNightmaree/.L/repo/luvi-master/build/openssl/src/openssl-stamp/openssl-build-*.log


gmake[3]: *** [CMakeFiles/openssl.dir/build.make:87: openssl/src/openssl-stamp/openssl-build] Error 1
gmake[3]: Leaving directory '/home/UrNightmaree/.L/repo/luvi-master/build'
gmake[2]: *** [CMakeFiles/Makefile2:182: CMakeFiles/openssl.dir/all] Error 2            gmake[2]: *** Waiting for unfinished jobs....
[ 62%] Linking C static library libluajit.a
gmake[3]: Leaving directory '/home/UrNightmaree/.L/repo/luvi-master/build'
[ 62%] Built target luajit-5.1
gmake[2]: Leaving directory '/home/UrNightmaree/.L/repo/luvi-master/build'              gmake[1]: *** [Makefile:156: all] Error 2
gmake[1]: Leaving directory '/home/UrNightmaree/.L/repo/luvi-master/build'
make: *** [Makefile:60: luvi] Error 2
```

**Comment by sumeetpri** [comment]
@t8m @UrNightmaree  - I am also facing same issue in ubantu image . What could be solution to this ?

**Comment by faramir-dev** [comment]
@UrNightmaree @sumeetpri Could you share ` configdata.pm`? It is generated at the beginning of the build process.

**Comment by nhorman** [comment]
marking as inactive, will be closed at the end of the 3.4 dev cycle barring further updates

**Comment by cattokomo** [comment]
I have switched to EndavourOS. If anyone have similar issue (especially on ARM Ubuntu), feel free to create new issue.
